static/app/views/issueList/overview.tsx

```diff
@@ -65,6 +65,7 @@ import IssueListFilters from './filters';
 import GroupListBody from './groupListBody';
 import IssueListHeader from './header';
 import {
+  DEFAULT_ISSUE_STREAM_SORT,
```

Comment:
These are mostly unrelated changes right?

Reply:
I needed to know what the default sort was in `IssueListSetAsDefault`, so I moved that constant to the utils file

Reply:
Oh I see cool